Source/Mosa.Templates/templates/Mosa.Starter.x86/Program.cs

```diff
@@ -1,16 +1,16 @@
 ﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
 
-using System;
+using Mosa.Kernel.x86;
 
 namespace Mosa.Starter.x86
 {
     public static class Program
     {
         public static void Setup()
         {
-            Console.ResetColor();
-            Console.Clear();
-            Console.WriteLine("Hello World!");
+            Screen.ResetColor();
```

Comment:
change back to using console

Reply:
This means the Console plug will have to be implemented in the template, is it fine?